instrumentation/build.gradle.kts

```diff
@@ -13,7 +13,9 @@ subprojects {
         implementation("net.bytebuddy:byte-buddy:${versions["byte_buddy"]}")
 
         implementation("io.opentelemetry:opentelemetry-api:${versions["opentelemetry"]}")
-        implementation("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling:${versions["opentelemetry_java_agent"]}")
+        implementation("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling:${versions["opentelemetry_java_agent"]}") {
+            exclude(group = "io.netty")
```

Comment:
it seems that we cannot use exclude. It breaks uber jar functionality -  the gRPC exporters will not work